sopel/modules/seen.py

```diff
@@ -36,17 +37,18 @@ def seen(bot, trigger):
         tz = get_timezone(bot.db, bot.config, None, trigger.nick,
                           trigger.sender)
         saw = datetime.datetime.utcfromtimestamp(timestamp)
+        delta = seconds_to_human((trigger.time - saw).total_seconds())
         timestamp = format_time(bot.db, bot.config, tz, trigger.nick,
                                 trigger.sender, saw)
 
-        msg = "I last saw {} at {}".format(nick, timestamp)
+        msg = "I last saw " + nick
         if Identifier(channel) == trigger.sender:
             if action:
-                msg = msg + " in here, doing " + nick + " " + message
+                msg = msg + " in here " + delta + ", doing: " + nick + " " + message
             else:
-                msg = msg + " in here, saying " + message
+                msg = msg + " in here " + delta + ", saying: " + message
         else:
-            msg += " in another channel."
+            msg += " in another channel " + delta + "."
         bot.say(str(trigger.nick) + ': ' + msg)
```

Comment:
If you fancy it, you can upgrade the code with `bot.reply`:

```suggestion
         bot.reply(msg)
```